examples/tutorial-examples/CMakeLists.txt

```diff
@@ -13,6 +13,10 @@ SET(DGTAL_TUTO_EXAMPLES_SRC
 
 
 FOREACH(FILE ${DGTAL_TUTO_EXAMPLES_SRC})
+  if(MSVC)
+    set_source_files_properties( ${FILE} PROPERTIES COMPILE_FLAGS "/bigobj" )
```

Comment:
In not familiar with MS builds. Why do you need to add the `/bigbuild`?
If it is required, do we need to add this to all targets (examples, tests, dgtaltools...) ?

Reply:
this flag is only needed by the example project "shortcuts-geometry" in the debug configuration. When compiling, the error is : https://docs.microsoft.com/en-us/cpp/error-messages/compiler-errors-1/fatal-error-c1128?view=vs-2017
The fix could have been, but too much specific for my point of view  : 
```
FOREACH(FILE ${DGTAL_TUTO_EXAMPLES_SRC})
  add_executable(${FILE} ${FILE})
  target_link_libraries (${FILE} DGtal  )
ENDFOREACH(FILE)

if(MSVC)
  set_source_files_properties( shortcuts-geometry PROPERTIES COMPILE_FLAGS "/bigobj" )
endif (MSVC)
```

Reply:
ok.. Has this issue only been raised for this shortcut example ?

Reply:
Yes, this was notified by @BorisMansencal in this issue : https://github.com/DGtal-team/DGtal/issues/1381

Reply:
👌